src/screens/ModelPage/ModelPage.js

```diff
@@ -42,19 +42,49 @@ const LastDatesToAct = ({ model, outOfBoundsDate }) => {
     return `${month} ${day}`;
   }
 
+  function suggestedIntervention() {
+    switch (intervention) {
+      case INTERVENTIONS.NO_ACTION:
+        return INTERVENTIONS.SHELTER_IN_PLACE;
+      case INTERVENTIONS.SOCIAL_DISTANCING:
+        return INTERVENTIONS.SHELTER_IN_PLACE;
+      case INTERVENTIONS.SHELTER_IN_PLACE:
+        // TODO: Are we okay recommending lockdown?
```

Comment:
$0.02 is to hold off on this. Shelter in place seems to give the needed room in capacity.

Reply:
I dropped it.  NY is the closest to possibly hitting this state (by our model).  If they do, they'll see "stricter intervention" and we can revisit.